modules/base/include/modules/base/algorithm/randomutils.h

```diff
@@ -346,11 +350,15 @@ std::shared_ptr<Image> poissonDisk(size2_t dims, size_t poissonDotsAlongX, size_
         return glm::i32vec2(newX, newY);
     };
 
-    auto gridImg = std::make_unique<Image>(gridSize, DataVec2Int32::get());
-    auto grid = gridImg->getColorLayer()->getEditableRepresentation<LayerRAM>();
+    std::shared_ptr<Layer> layer =
```

Comment:
create the LayerRAMPrecision first and add that to the layer...

or retur the LayerRAMPrecision<float> instead